src/data/python-quiz.ts

```diff
@@ -1581,6 +1581,15 @@ const pythonQuiz = [
     Explanation: "In Python, math.trunc() will return the truncated integer part of the number. When the number is positive, math.trunc() is similar to the .floor() method. But if the number is negative, math.trunc() is similar to the .ceil() method",
     Link: "https://docs.python.org/3/library/math.html",
   },
+   {
+    Question: "In Python, what will be the output of the following code snippet? lst = [10, 20, 30, 40, 50, 60, 70, 80, 90] print(lst[:5:2])?",
```

Comment:
long questions don't work well when rendered to the page
```suggestion
    Question: "In Python, what is the output for the following: print([10, 20, 60, 90][:2:])?",
```